.travis.yml

```diff
@@ -6,8 +6,18 @@ rvm:
   - 2.5.3
   - 2.6.1
   - jruby-9.1.10.0
+  - jruby-9.2.7.0
 gemfile:
   - Gemfile
   - Gemfile.rails-3.2
   - Gemfile.rails-4.2
   - Gemfile.rails-5.2
+  - Gemfile.rails-6.0
+matrix:
+  exclude:
```

Comment:
why are these `exclude` here?

Reply:
rails six does not run on 2.5 and this older jruby.

Reply:
> JRuby 9.2.x is compatible with Ruby 2.5.x and stays in sync with C Ruby.
> — https://www.jruby.org/2019/08/12/jruby-9-2-8-0.html

it‘s not mentioned in the rails 6 release notes but [this](https://github.com/rails/rails/pull/34754).

Reply:
thanks, wasn't aware of that.